src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -3353,7 +3353,7 @@ namespace InternalHeatGains {
 
 				//For predefined tabular reports related to outside air ventilation
 				ZonePreDefRep( NZ ).isOccupied = true; //set flag to occupied to be used in tabular reporting for ventilation
-				ZonePreDefRep( NZ ).NumOccAccum += NumberOccupants *  Zone( NZ ).Multiplier * Zone( NZ ).ListMultiplier * TimeStepZone;
+				ZonePreDefRep( NZ ).NumOccAccum += NumberOccupants * TimeStepZone;
```

Comment:
@rraustad Surprised to see the multipliers removed here.  The OA flow rates are all multiplied, so I would expect the number of occupants to be multiplied to be consistent for the report.